### DISCUSSION TITLE
Saving simulation

### DISCUSSION THREAD

**Discussion by Jadhav-Sourabh** [pybamm-team/PyBaMM#2145] [General]
can someone please explain to me how to save the simulation in one file? so that I will use that file again only not the simulation.

**Comment by brosaplanella** [comment]
[This notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Getting%20Started/Tutorial%206%20-%20Managing%20simulation%20outputs.ipynb) explains how to save simulations into files.

**Reply by Jadhav-Sourabh** [reply]
Thank you very much :)
